### ISSUE TITLE
cannot use $, $1 , etc... in ExtendedDescription

### ISSUE THREAD

**Issue by sarahBuisson** [TNG/JGiven#236]
I use jgiven 0.11.4

```
@As("Insert the $2 into the bucket")
@ExtendedDescription("put the file of $1 into the bucket folder $2")
public WhenEnd2end put_files_to_s3(String importPath, String bucketFolder) {
```

report:

>  Insert the products into the bucket products
>  put the file of $1 into the bucket folder $2

**Comment by Airblader** [comment]
This relates to and depends on #237 and would involve changing the DefaultAsProvider

**Comment by janschaefer** [comment]
fixed by #283
